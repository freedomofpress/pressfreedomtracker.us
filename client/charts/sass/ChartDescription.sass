.chartDescription
	height: 2rem
	padding: 0.75rem
	font-family: var(--font-base)
	font-weight: 400
	font-size: 1rem
	color: #757575
	margin-bottom: 1rem
