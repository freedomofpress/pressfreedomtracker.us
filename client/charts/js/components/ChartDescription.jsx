import React from 'react'
import '../../sass/ChartDescription.sass'

export default function ChartDescription({ id, children }) {
	return (
		<div
			id={id}
			className='chartDescription'
		>
			{children}
		</div>
	)
}
