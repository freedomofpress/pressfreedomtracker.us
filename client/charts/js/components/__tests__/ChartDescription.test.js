import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import ChartDescription from '../ChartDescription'

test('renders ChartDescription with mocked data', () => {
	const renderer = new ShallowRenderer();
	expect(renderer.render(
		<ChartDescription id="test">
			This is a test
		</ChartDescription>
	)).toMatchSnapshot();
});
