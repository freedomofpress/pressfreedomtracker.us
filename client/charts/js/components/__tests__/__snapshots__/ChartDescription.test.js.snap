// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ChartDescription with mocked data 1`] = `
<div
  className="chartDescription"
  id="test"
>
  This is a test
</div>
`;
